edenia/antelope-tools
# Feat/improve bpjson health check 1004

### Use the same icons for health check indicator

### What does this PR do?

- Resolve #1004
- Resolve #1018 
- Fix a bug in block producers page when the node_type is undefined
- Remove the "nodes" key from bpjsons that are not from the current network. 

### Steps to test

1. Run the project locally
2. Check to block producers page
3. Check to nodes page
4. Check the endpoints page

### Screenshots

#### Block producers
![imagen](https://user-images.githubusercontent.com/66583677/200073985-59d976e7-24e5-4506-b488-0551ca0960d8.png)

#### Nodes

![imagen](https://user-images.githubusercontent.com/66583677/200074050-b895704a-b011-4c04-b67b-be51a6336c32.png)

#### Endpoints

![imagen](https://user-images.githubusercontent.com/66583677/200074082-986a62e6-c9e0-4a5e-a304-b5f01c7282d3.png)
